cypress/utils/data_utils.ts

```diff
@@ -15,3 +15,15 @@ export function getBusinessServiceName(): string {
 export function getBusinessServiceDescription(): string {
     return faker.lorem.sentence();
 }
+
+export function getApplicationName(): string {
+    return faker.name.title();
+}
+
+export function getApplicationDescription(): string {
```

Comment:
Create a unique method , getDescription() and getComment() . to be used in all models .